docs/releases/overview.md

```diff
@@ -26,6 +26,25 @@ See the full release notes on [GitHub ↗](https://github.com/vegaprotocol/vega/
 ## Vega core software
 The Vega core software is public on a business-source licence, so you can both view the repository change logs, and refer here for summary release notes for each version that the validators use to run the Vega mainnet. Releases are listed with their semantic version number and the date the release was made available to mainnet validators.
 
+### Pre-release Versions 0.70.0 and 0.70.1 combined | 2023-03-28
+This version was released to the Vega testnet on 28 March 2023.
+
+Version 0.70.0 and 0.70.1 contains the fixes and minor enhancements to verify in testnet before the validators deploy for the Market Simulation #4.
+
+This deployment realises a number of further data node enhancements to aid performance and the management of the stored data. The indexes on the positions table have been reworked in order to maintain the performance of the network history on the data nodes. Additionally, the config for the size of the buffer has been adjusted to best utilise the node memory on startup. Finally, to help manage the volume of data being created, LP orders are no longer sent when resubmitted without any changes, giving a direct data storage benefit.
+
+:::caution Breaking changes
+**Add `marketIds` and `partyIds` to orders queries' filter.**: In order to allow getting all orders for a single party or market so that users can more easily find their orders across multiple keys or markets the API filtering has been enhanced.
```

Comment:
```suggestion
**Add `marketIds` and `partyIds` to orders queries' filter.**: In order to allow getting all orders for a single party or market so that users can more easily find their orders across multiple keys or markets, filtering on the orders endpoint has been enhanced.
```